chrysom/client_test.go

```diff
@@ -196,7 +209,7 @@ func TestSendRequest(t *testing.T) {
 			}
 
 			assert.Nil(err)
-			resp, err := client.sendRequest(tc.Owner, tc.Method, URL, bytes.NewBuffer(tc.Body))
+			resp, err := client.sendRequest(tc.Owner, tc.Method, URL, bytes.NewBuffer(tc.Body),context.TODO())
```

Comment:
Could we add a sample context here with trace info so we could assert on the otel trace headers being injected into the request?

Reply:
We will not have TraceMiddileware here which is responsible for starting the new span with a new/existing traceID. if that was here then we can do the testing of headers here.